common/src/main/java/org/vivecraft/client_vr/provider/openxr/MCOpenXR.java

```diff
@@ -802,20 +798,20 @@ private void initializeOpenXRInstance() {
                 }
             }
 
-            if (missingOpenGL) {
-                throw new RuntimeException("OpenXR runtime does not support OpenGL, try using SteamVR instead");
+            if (missingGraphics) {
+                throw new RuntimeException("OpenXR runtime is missing a supported graphics extension.");
             }
 
             //Create APP info
             XrApplicationInfo applicationInfo = XrApplicationInfo.calloc(stack);
-            applicationInfo.apiVersion(XR10.XR_CURRENT_API_VERSION);
+            applicationInfo.apiVersion(XR10.XR_MAKE_VERSION(1, 0 , 14));
```

Comment:
Why force openxr 1.0.14 here?

Reply:
> Why force openxr 1.0.14 here?

This is mostly done to ensure the runtime used is compatible between both platforms, though I can add another specifier in the DeviceCompat interface for that instead.

Reply:
I think it's fine to use a fixed version, to not have something break because of spec changes
just curious why exactly that one,. is it the olders one that works?

Reply:
I believe 1.0.14 is the oldest one that works without sacraficing certain API usage, and 1.0.40 is the newest that can be used without the input path being changed (good for us as we want to ensure full path to avoid action conflicts).